### PR TITLE
Add LoginMode property in OktaMvcOptions 

### LINKED ISSUE #77: Self-hosted samples redirect to Okta instead of the self-hosted login
When you are not authenticated and try to access a controller decorated with `[Authorize]` attribute, it redirects to Okta instead of the local sign in page.

### PR DESCRIPTION
**Pull request by laura-rodriguez**
Fixes #77 

This property is only available on ASP.NET since ASP.NET Core already has a workaround: https://github.com/okta/samples-aspnetcore/pull/27.

### CODE REVIEW

File: Okta.AspNet/LoginMode.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="LoginMode.cs" company="Okta, Inc">
+// Copyright (c) 2018-present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace Okta.AspNet
+{
+     /// <summary>
+     /// LoginMode controls the behavior of the middleware.
```
Comment (OkGoDoIt) [inline Okta.AspNet/LoginMode.cs]: I would elaborate here, maybe something like "LoginMode controls the login redirect behavior of the middleware"

File: Okta.AspNet/OktaMvcOptions.cs [hunk 2]
```diff
@@ -44,6 +44,12 @@ public class OktaMvcOptions : Abstractions.OktaWebOptions
         /// <value>The scope.</value>
         public IList<string> Scope { get; set; } = OktaDefaults.Scope;
 
+        /// <summary>
+        /// Gets or sets the <see cref="LoginMode"/>.
```
Comment (OkGoDoIt) [inline Okta.AspNet/OktaMvcOptions.cs]: I would elaborate here, maybe something like "Controls the login behavior to redirect to internal or Okta-hosted login"

**Comment by OkGoDoIt** [comment]
So can this LoginMode property be set in the normal Web.config way too?   Have you tested to make sure the enum is parsed properly?  Maybe the docs need to be updated to list this as well?

**Comment by laura-rodriguez** [comment]
@OkGoDoIt Yes, you can set this property in the `Web.Config` as in this example:https://stackoverflow.com/questions/8148622/get-enum-values-from-web-config-at-run-time.
I don't think we need to test if this is parsed correctly since that's not the responsibility of the middleware.
Docs definitely need to be updated, but I wanted to hear the team's thoughts first.

**Comment by OkGoDoIt** [comment]
I would personally prefer to update the method summary in the second location I commented on above. And are you absolutely certain the web config is parsed correctly for the enum? The link you posted makes it look like you have to do it manually, but I don’t see any code on our end to handle that.

**Comment by laura-rodriguez** [comment]
Yeah, we don't that. They way to do that in this SDK is via `Configuration` object.

**Pull request merged**